types/object-fit-images/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for object-fit-images 3.2
+// Project: https://github.com/bfred-it/object-fit-images#readme
+// Definitions by: Ranjit Singh <https://github.com/rann91>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function objectFillImages(
+    images: string | HTMLElement | HTMLElement[] | NodeList | null,
+    options?: { watchMQ: boolean },
```

Comment:
There seems to be undocumented `skipTest` option (also bool):
https://github.com/fregante/object-fit-images/blob/master/index.js#L182